providers/org/biorxiv/normalizer.py

```diff
@@ -1,22 +1,37 @@
-from share.normalize import *  # noqa
+from share.normalize import links
+from share.normalize.parsers import Parser
+
+ctx = links.Context()
+
+
+class Publisher(Parser):
+    name = ctx
+
+
+class Association(Parser):
+    pass
 
 
 class Person(Parser):
-    given_name = ParseName(ctx).first
-    family_name = ParseName(ctx).last
-    additional_name = ParseName(ctx).middle
-    suffix = ParseName(ctx).suffix
+    given_name = links.ParseName(ctx).first
+    family_name = links.ParseName(ctx).last
+    additional_name = links.ParseName(ctx).middle
+    suffix = links.ParseName(ctx).suffix
 
 
 class Contributor(Parser):
-    order_cited = ctx['index']
-    person = ctx
+    order_cited = ctx('index')
+    person = links.Delegate(Person, ctx)
     cited_name = ctx
 
 
 class CreativeWork(Parser):
-    title = ctx.item('dc:title')
+    title = ctx.item['dc:title']
     description = ctx.item.description
-    contributors = ctx.item('dc:creator')['*']
-    published = ctx.item('dc:date')
-    doi = ctx.item('dc:identifier')
+    contributors = links.Map(links.Delegate(Contributor), ctx.item['dc:creator'])
+    published = ctx.item['dc:date']
+    publishers = links.Map(
+        links.Delegate(Association.using(entity=links.Delegate(Publisher))),
+        ctx.item['dc:publisher']
+    )
+    # doi = ctx.item['dc:identifier']
```

Comment:
Create a parser for `Link` and `ThroughLink` 
https://github.com/CenterForOpenScience/SHARE/blob/develop/share/models/meta.py#L42
I'll make the `ThroughLink` model in a second